tasks/fix-cat2.yml

```diff
@@ -1292,16 +1292,31 @@
     - {id: "030874", path: "/etc/security/opasswd", description: "account creations, modifications, disabling, and termination events that affect /etc/security/opasswd", key: "identity", create: "{{ rhel_07_030874 }}"}
 
 - name: "MEDIUM | RHEL-07-031000 | PATCH | The system must send rsyslog output to a log aggregation server."
-  command: "true"
-  changed_when: no
-  when: rhel_07_031000
+  blockinfile:
+    path: /etc/rsyslog.conf
+    state: present
+    block: |
+      $ActionQueueFileName rhel-07-031000 # unique name prefix for spool files
+      $ActionQueueMaxDiskSpace 1g   # 1gb space limit (use as much as possible)
+      $ActionQueueSaveOnShutdown on # save messages to disk on shutdown
+      $ActionQueueType LinkedList   # run asynchronously
+      $ActionResumeRetryCount -1    # infinite retries if host is down
+      # remote host is: name/ip:port, e.g. 192.168.0.1:514, port optional
+      *.* @@{{ rhel7stig_log_aggregation_server }}
```

Comment:
I  think if you add ' | default("") to this variable, it will fix the error.

Reply:
The weird thing is the idempotence test is what's failing, [due to ssh key changes](https://travis-ci.org/MindPointGroup/RHEL7-STIG/builds/357455111#L4066) that should have nothing to do with this commit.

Reply:
ok, i didn't actually look at the output...